README.md

```diff
@@ -46,6 +51,13 @@ You can handle refunds using the following constructs:
 
 ```rb
 Razorpay::Payment.fetch("payment_id").refund({amount:500})
+refunds = Razorpay::Payment.fetch("payment_id").refunds
+```
+
+Refunds can also be handled without fetching payments:
+```rb
+refund = Razorpay::Refund.create(payment_id:"payment_id")
```

Comment:
Yes, this is the new style, and I'd prefer `symbol: "Value"`. The old style is in a few other places, and we can change it elsewhere.